### PR TITLE
release notes for 06 dec staging release

### PR DESCRIPTION
**Pull request by leysanchez**

### CODE REVIEW

File: docs/whats-new/staging-release-notes.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
 
+## December 2023
+
+**06 December 2023**
+
+Frontend version: 1.0.0-20231130.0955 | Backend version: 1.78.2-20231201.1059
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Change** | Get Existing Users API now checks account existence in the source AAD | Previously, the **Get Existing Users API** implementation excluded orphaned accounts that have been terminated in WoG, but not in TechPass. We modified this API to check if an account exists in the source AAD. For members and users from an unsupported directory, it checks TechPass AAD. For guest accounts from a supported directory, it checks the parent directory.
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md]:
```suggestion
| **Change** | Get Existing Users API now checks account existence in the source Entra ID | Previously, the **Get Existing Users API** implementation excluded orphaned accounts that have been terminated in WoG, but not in TechPass. We modified this API to check if an account exists in the source Entra ID. It checks TechPass Entra ID for members and users from an unsupported directory. It checks the parent directory for accounts from an integrated Entra ID directory
```

File: docs/whats-new/staging-release-notes.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+| **Change** | New SEED status for TechPass users ineligible for SEED | We've introduced a new **Ineligible** status for users who don't qualify for SEED, such as non-native TechPass users not in WoG AAD. Previously, these users were labeled as **Not Required**, which is intended for users without an assigned license.
```
Comment (yilunyou) [inline docs/whats-new/staging-release-notes.md]:
```suggestion
| **Change** | New SEED status for TechPass users ineligible for SEED | We've introduced a new **Ineligible** status for users who don't qualify for SEED, such as orphaned accounts or accounts from integrated IDPs that doesn't require SEED (ie. MOE). Previously, these users were labeled as **Not Required**, which is intended for users without an assigned license.
```

Comment (yilunyou) [reply docs/whats-new/staging-release-notes.md]: what do u mean by non-native TechPass users?

Comment (leysanchez) [reply docs/whats-new/staging-release-notes.md]: > what do u mean by non-native TechPass users?

I got the term from the JIRA story (https://gdsjira.ship.gov.sg/browse/TECHPASS-3564). Maybe @yqyeoh can provide some context? And can you recommend a better term?

File: docs/whats-new/staging-release-notes.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
 
+## December 2023
+
+**06 December 2023**
+
+Frontend version: 1.0.0-20231130.0955 | Backend version: 1.78.2-20231201.1059
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Change** | Get Existing Users API now checks account existence in the source Entra ID | Previously, the **Get Existing Users API** implementation excluded orphaned accounts that have been terminated in WoG, but not in TechPass. We modified this API to check if an account exists in the source Entra ID. It checks TechPass Entra ID for members and users from an unsupported directory. It checks the parent directory for accounts from an integrated Entra ID directory
```
Comment (yqyeoh) [inline docs/whats-new/staging-release-notes.md]:
```suggestion
| **Change** | Get Existing Users API now checks account existence in the source Entra ID | Previously in **Get Existing Users API**, orphaned accounts that are not found in the source Entra ID (e.g. WoG) were included in the response. We have modified this API to verify the existence of an account in the source Entra ID and exclude it from the response if it is not found. 
```

**Review by yilunyou — APPROVED** [review]

**Pull request merged**